ansible/group_vars/all

```diff
@@ -139,6 +139,7 @@ registry:
   confdir: "{{ config_root_dir }}/registry"
 
 kafka:
+  topicsPrefix: "{{ kafka_topics_prefix | default('') }}"
```

Comment:
Add prefix for kafka topic is implemented in this pr: https://github.com/apache/openwhisk/pull/5062